src/activities/Elsa.Activities.Mqtt/Elsa.Activities.Mqtt.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <RootNamespace>Elsa.Activities.Mqtt</RootNamespace>
+        <LangVersion>latest</LangVersion>
+        <Authors>Elsa Contributors</Authors>
+        <Description>Elsa is a set of workflow libraries and tools that enable lean and mean workflowing capabilities in any .NET Core application.
+This package provides activities to send and receive messages using MQTT.</Description>
+        <Copyright>2021</Copyright>
+        <PackageProjectUrl>https://github.com/elsa-workflows/elsa-core</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/elsa-workflows/elsa-core</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>elsa, workflows</PackageTags>
+    </PropertyGroup>
```

Comment:
Most of these elements can be included via common.props. Take a look at one of the other .csproj files to see what props to include.

Reply:
Added common.props, thanks for the tip!